.github/workflows/e2e-browser.yml

```
@@ -28,40 +32,38 @@ jobs:
         env:
           cache-name: cache-node-modules
         with:
-          path: 
-            node_modules
+          path: node_modules
             packages/*/node_modules
           key: ${{ runner.os }}-node${{ runner.node-version }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
       - run: npm ci
       - run: npm run bootstrap -- --ci
       - run: npm run build
+      - name: Install e2e tests dependencies
+        run: |
+          cd e2e/browser/
+          npm ci
+          npx playwright install-deps chromium firefox webkit
+          npx playwright install
```

```suggestion
          npx playwright install --with-deps
```

This does the same as the above, but with a single step.

---

Fixed in 068a5b49